tests/seed_isort_config_test.py

```diff
@@ -51,7 +52,10 @@ def test_integration_isort_cfg(tmpdir):
         assert tmpdir.join('.isort.cfg').read() == expected
 
 
-@pytest.mark.parametrize('filename', ('setup.cfg', 'tox.ini'))
+@pytest.mark.parametrize(
+    'filename',
+    [k for k, v in CONFIG_FILE_TO_START_RE.items() if k != '.isort.cfg'],
```

Comment:
no need for `.items()`, you're not using the value -- easier: `set(CONFIG_FILE_START) - {'.isort.cfg'}`

Reply:
Good catch